### PR TITLE
Paf 1

### PR DESCRIPTION
**Pull request by neopaf**

### CODE REVIEW

**Comment by neopaf** [comment]
Please consider dropping leading '...' in shortened paths. It helped me immensely!

Prior to this, Lua 5.1 on Mac failed and stack trace contained long paths shortened.

I'm not sure how to create PRs, this is my first. So there's lots of extra junk. Effectively we need one last commit

**Comment by CppCXY** [comment]
This PR will not be received for a long time, it is recommended that you use your own compiled plugin

File: gradle.properties [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-version = 1.3.6
+version = 1.3.6-paf1
```
Comment (CppCXY) [inline gradle.properties:17]: It should not be committed

Comment (neopaf) [reply gradle.properties:17]: Absolutely.
That should have been on some other branch.
Please just consider using 1 line change, just yourself?

**Pull request closed without merging**